test/integration/api/admin/api_spec.js

```diff
@@ -425,6 +427,51 @@ describe("api", function () {
         }));
       });
     });
+
+    describe('selective response', () => {
+      const expectedKeys = ['public_id', 'asset_id', 'folder', 'tags'].sort();
+
+      it('should allow listing', async () => {
+        const {resources} = await cloudinary.v2.api.resources({fields: ['tags']})
+        const actualKeys = Object.keys(resources[0]);
```

Comment:
There is a lot of repetition here so worth considering adding a helper like `assertKeysEqual(resource[0])` which would do all the `Object.keys` and `deepStrictEqual` and `sort`ing inside of it. 

That would probably make the tests more readable too. Not that they are somehow very complicated, just a thought.